tests/integration/test_logcollector/test_configuration/test_basic_configuration_reconnect_time.py

```diff
@@ -0,0 +1,118 @@
+    {'log_format': 'eventchannel', 'location': f'{location}', 'reconnect_time': 'Testing44s', 'valid_value': False},
+    {'log_format': 'eventchannel', 'location': f'{location}', 'reconnect_time': '9hTesting', 'valid_value': False},
+    {'log_format': 'eventchannel', 'location': f'{location}', 'reconnect_time': '400mTesting', 'valid_value': False},
+    {'log_format': 'eventchannel', 'location': f'{location}', 'reconnect_time': '3992', 'valid_value': False},
+    {'log_format': 'eventchannel', 'location': f'{location}', 'reconnect_time': 'Testing', 'valid_value': False},
+]
+
+configurations = load_wazuh_configurations(configurations_path, __name__,
+                                           params=parameters,
+                                           metadata=metadata)
+configuration_ids = [f"{x['LOG_FORMAT'], x['LOCATION'], x['RECONNECT_TIME']}" for x in parameters]
+problematic_values = ['44sTesting', '9hTesting', '400mTesting', '3992']
+
+
+def check_configuration_reconnect_time_valid():
+    """Check if the Wazuh module run correctly and that analyze the desired eventchannel.
+
+    Ensure logcollector is running with the specified configuration, analyzing the designate eventchannel.
+
+
+    Raises:
+        TimeoutError: If the "Analyzing eventchannel" callback is not generated.
+    """
+
+    log_callback = logcollector.callback_eventchannel_analyzing('Security')
+    wazuh_log_monitor.start(timeout=5, callback=log_callback,
+                            error_message="The expected error output has not been produced")
+
+
+def check_configuration_reconnect_time_invalid(cfg):
+    """Check if the Wazuh fails because a invalid reconnect time attribute configuration value.
+
+    Args:
+        cfg (dict): Dictionary with the localfile configuration.
+
+    Raises:
+        TimeoutError: If error callback are not generated.
+    """
+    if cfg['reconnect_time'] in problematic_values:
+        pytest.xfail("Logcolector accepts invalid values. Issue: https://github.com/wazuh/wazuh/issues/8158")
+
+    log_callback = logcollector.callback_invalid_reconnection_time(prefix=prefix)
+    wazuh_log_monitor.start(timeout=5, callback=log_callback,
+                            error_message="The expected invalid reconnection time error has not been produced")
+
+
+# fixtures
+@pytest.fixture(scope="module", params=configurations, ids=configuration_ids)
+def get_configuration(request):
+    """Get configurations from the module."""
+    return request.param
+
+
+def test_configuration_reconnect_time(get_configuration, configure_environment, restart_logcollector):
+    """Check if the Wazuh reconnect time field of logcollector works properly.
+
+    Ensure Wazuh component fails in case of invalid values and works properly in case of valid reconnect time values.
+
+    Raises:
+        TimeoutError: If expected callbacks are not generated.
```

Comment:
```suggestion
    """Check if Wazuh reconnects time field of logcollector works properly.

    Ensure Wazuh component fails in case of invalid values and works properly in case of valid reconnect time values.

    Raises:
        TimeoutError: If expected callbacks are not generated.
```